_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,407 @@
+---
+id: 9bd38c
+name: Non-visual reference alternative
+rule_type: atomic
+description: |
+  This rule checks that when there is a visual reference of content, there are also non-visual indicators of the location.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
```

Comment:
This rule only checks HTML. And G96 is a general technique that applies to all technologies. So you can have a situation where this rule passes (because all the HTML checks out) but on the same page you have an embedded PDF or something that fails this technique.